types/mongoose/index.d.ts

```diff
@@ -1937,7 +1938,7 @@ declare module "mongoose" {
      * getters/setters or other Mongoose magic applied.
      * @param bool defaults to true
```

Comment:
Does the parameter type in the JSDoc need to be updated as well?

Reply:
Thanks for the reminder. I changed it to `@param {Boolean|Object} bool defaults to true` to alight with https://github.com/Automattic/mongoose/blob/master/lib/query.js#L1636